tests/common/devices/sonic.py

```diff
@@ -2317,6 +2317,17 @@ def links_status_up(self, ports):
                 logging.info("Interface {} is up on {}".format(output_port, self.hostname))
         return True
 
+    def interface_status_up(self, interface):
```

Comment:
**is_**interface_status_up would be more clear

Reply:
Done